lib/plugins/aws/provider.js

```diff
@@ -1806,6 +1846,17 @@ class AwsProvider {
       );
     }
   }
+
+  reportDeprecatedUsage(deprecationCode, pairs) {
+    for (const [deprecated, modern] of pairs) {
+      if (_.has(this.serverless.service.provider, deprecated)) {
```

Comment:
Let's not use `_.has` (we can do `this.serverless.service.provider[deprecated] != null`)

Reply:
If you want to make it an util, I'd make props like `provider.role` etc so it's also usable for deprecating function props and so on. This will require usage of `_.get` or `_.has` Thoughts?

Reply:
I think indeed that `_.has` is ok in that case